cache-aside/README.md

```diff
@@ -3,8 +3,8 @@ Copyright 2019 - 2021 VMware, Inc.
 SPDX-License-Identifier: Apache-2.0
 -->
 
-# Look-aside Cache Pattern Example
+# Cache-aside Pattern Example
```

Comment:
Should we capitalize the 'a' here? I realize it wasn't capitalized before, but it probably should've been.

Reply:
Yes, good catch.